odoo/tools/config.py

```diff
@@ -416,7 +416,7 @@ def die(cond, msg):
             elif isinstance(self.options[arg], basestring) and self.casts[arg].type in optparse.Option.TYPE_CHECKER:
                 self.options[arg] = optparse.Option.TYPE_CHECKER[self.casts[arg].type](self.casts[arg], arg, self.options[arg])
 
-        self.options['root_path'] = os.path.abspath(os.path.expanduser(os.path.expandvars(os.path.dirname(odoo.__file__))))
+        self.options['root_path'] = os.path.abspath(os.path.expanduser(os.path.expandvars(os.path.join(os.path.dirname(__file__), '..'))))
```

Comment:
With namespace package, the same `odoo` package can have source in different places on the file system. For instance it can be the main odoo package but also the root of some random odoo/addon/someaddon residing elsewhere on disk. 

I think @lmignon has met that issue when doing preliminary testing last year.

`dirname(__file__)/..` OTOH is sure to point to the right directory in all cases. So it's safer.